spec/graphql/queries/dog_spec.rb

```diff
@@ -9,45 +9,26 @@
     json = JSON.parse(response.body)
     data = json['data']['dog']
 
-    expect(data).to include(
-      'id'         => dog.id.to_s,
-      'name'       => dog.name,
-      'breed'      => dog.breed,
-      'weight'     => dog.weight,
-      'birthdate'  => dog.birthdate.to_s,
-      'shortDesc'  => dog.short_desc,
-      'longDesc'   => dog.long_desc
-    )
-
+    compare_gql_and_db_dogs(data, dog)
+    
     actual_user = data['user']
-    expect(actual_user).to include(
-      'id'          => user.id.to_s,
-      'firstName'   => user.first_name,
-      'lastName'    => user.last_name,
-      'email'       => user.email,
-      'shortDesc'   => user.short_desc,
-      'longDesc'    => user.long_desc
-    )
+    compare_gql_and_db_users(actual_user, user)
+  end
+
+  it 'raises exception if no dog with that id' do
```

Comment:
This is interesting.  I'm still having a hard time convincing myself that we're testing anything here!